### ISSUE TITLE
Appveyor build got broken

### ISSUE THREAD

**Issue by bestander** [yarnpkg/yarn#3567]
https://ci.appveyor.com/project/kittens/yarn/build/2493 - Appveyor got red on this build and never recovered.

```
FAIL  __tests__\commands\add.js (132.432s)
  ● adds any new package to the current workspace, but install from the worktree
    Error: The worktree feature is currently experimental and needs to be manually enabled - please add "workspaces-experimental true" to your .yarnrc file.
      
      at MessageError (src\errors.js:5:5)
      at src\config.js:293:15
      at next (native)
      at step (node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
      at node_modules\babel-runtime\helpers\asyncToGenerator.js:28:13
      Console output:
       [1/4] Resolving packages...
      [2/4] Fetching packages...
      [3/4] Linking dependencies...
      [4/4] Building fresh packages...
      success Saved lockfile.
      success Folder in sync.
      success Folder in sync.
      
      at __tests__\commands\_helpers.js:144:13
      at throw (native)
      at step (node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
      at node_modules\babel-runtime\helpers\asyncToGenerator.js:30:13
  × adds any new package to the current workspace, but install from the worktree (22914ms)
  √ install with arg (22424ms)
  √ install from github

```

On other OS it passes.

**Comment by bestander** [comment]
cc @arcanis, without this we can't release 0.26, windows binaries won't build.

**Comment by arcanis** [comment]
@bestander For some reason my local tests also stopped passing - any idea?

<img width="1184" alt="screen shot 2017-06-05 at 11 47 30 am" src="https://cloud.githubusercontent.com/assets/1037931/26781121/dc9725e0-49e4-11e7-87bb-1335f0d00a34.png">

**Comment by bestander** [comment]
Are you testing on windows locally?

**Comment by arcanis** [comment]
Nope, that's the Linux tests. I'm testing on Windows via Appveyor. Worth noting that all tests seem to pass on CI, it's just my environment that breaks, but I have no idea what might cause these issues ...

**Comment by arcanis** [comment]
Will be fixed by #3578

**Comment by arcanis** [comment]
Fixed 👍
